app/test_main.py

```diff
@@ -1 +1,57 @@
-# write your code here
+from typing import Callable
+from unittest import mock
+from app.main import can_access_google_page
+
+import pytest
```

Comment:
Pytest should be in first block of imports, than blank line and than your own import